packages/common-sdk/src/web3/transactions/transactions-builder.ts

```diff
@@ -255,14 +255,16 @@ export class TransactionBuilder {
         (computeBudgetOption.priorityFeeLamports * MICROLAMPORTS_PER_LAMPORT) / computeLimit,
       );
 
-      prependInstructions = [
```

Comment:
I'm guessing both should be inside the if statement, but need to double check

Reply:
I think we need to always include computeUnitLimit but can optionally exclude computeLimitPrice if it is 0

Reply:
thanks for the comment. `fixed` now only includes `computeUnitPrice` if `microLamports` is greater than 0